Gruntfile.js

```diff
@@ -85,6 +86,12 @@ module.exports = function(grunt) {
             if (data.result.failures.length) {
                 failures[currentApp] = data.result.failures;
             }
+            if (grunt.option('coverage')) {
+                var coverageDir = './coverage-js/',
+                    filePath = coverageDir + currentApp.replace(/\//g, '-') + '.json';
+                if (!fs.existsSync(coverageDir)) { fs.mkdir(coverageDir, err => console.log(err)); }
```

Comment:
Is there cleanup of `coverageDir` at some point or is the idea to have this available after the cover report script is run?
Is `coverageDir` overwritten each time you run the script?

Reply:
The HTML version of the report ends up that directory and will want to stick around for sure. The json coverage files that are generated here could be useful for other tools but aren't strictly necessary to keep around. They'll get overwritten each time it runs, unless for some reason an app gets removed from the list of test apps (in which case that json file would just stay indefinitely). 

I could see a pre-run cleanup to make sure everything that makes it into the report is current -- something really simple like delete the directory if it exists, then run the script. Would that make sense?

Reply:
Yeah, I think some pre-cleanup would be good for keeping the directory concise and all of the files contained in it relevant to the current state of the js code.